torch/_torch_docs.py

```diff
@@ -4272,6 +4272,59 @@ def merge_dicts(*dicts):
     {out}
 """.format(**common_args))
 
+add_docstr(torch.xlogy,
+           r"""
+xlogy(x, y) -> Tensor
+
+Compute x*log(y) so that the result is 0 if x = 0.
```

Comment:
This is consistent with SciPy's documentation but misleading since xlogy(0, nan) is nan, not zero. 

Is that behavior an error, do you think? cc @rgommers. It would be interesting to look at this function's history to find out. It's a little odd that the following is true:

```
xlogy(0, nan) = 0 * log(nan) = 0 * nan = nan
xlogy(0, 0) = 0 * log(0) = 0 * -inf = 0
```

Since typically 0 * -inf is nan. So one case producing a nan is suppressed but another isn't.

Reply:
Definitely not an error, returning `nan` is correct and was tested in the PR that created `xlogy`: https://github.com/scipy/scipy/pull/469.

The point of the function is that `0 * log(0)` should return `0`, this is a common special case in, e.g., statistics functions (and a valid mathematical limit value). `nan`s however simply propagate.

If the nan case bothers you, maybe just tweak this line of the docstring to say:
```
Compute ``x*log(y)`` so that ``0 * log(0)`` returns zero.
```

Reply:
Thanks @rgommers for validating the current behavior, I agree we should tweak the docstring. Your proposal seems more precise.

Reply:
Updated to 
`Compute ``x*log(y)`` so that ``0 * log(0)`` returns zero.`